salt/deployment-manager/templates/dm-config.json.tpl

```diff
@@ -48,6 +48,7 @@
 {% set app_packages_hdfs_path = pillar['pnda']['app_packages']['app_packages_hdfs_path'] -%}
 
 {% set policy_file_link = pillar['resource_manager']['path'] + pillar['resource_manager']['policy_file'] %}
+{%- set flink_lib_dir = pillar['pnda']['homedir'] + '/flink-' + pillar['flink']['release_version'] + '/lib' -%}
```

Comment:
I think it would be better to create a symlink e.g. /opt/pnda/flink -> /opt/pnda/flink-1.2.3v5 when installing flink and then use that symlink. Otherwise when you upgrade flink you would have to re-run the deployment-manager sls.